src/diffusers/utils/testing_utils.py

```diff
@@ -54,6 +54,7 @@
 ) > version.parse("4.33")
 
 USE_PEFT_BACKEND = _required_peft_version and _required_transformers_version
+BIG_GPU_MEMORY = 40
```

Comment:
Would make configurable from the workflow file. 
```suggestion
BIG_GPU_MEMORY = os.getenv("BIG_GPU_MEMORY, 40)
```